docs/Installation Guide.md

```diff
@@ -27,7 +27,7 @@ If you don't already have git, you can download it [here](http://git-scm.com/dow
 
 Clone the project from GitHub by running the following command:
 
-    git clone project_url_here
+    git clone https://github.com/systers/vms
```

Comment:
I think this is unnecessary, as check the next line 

```
 For this project, it would correspond to:

```

Reply:
This change wasn't made by me. @tonythomas01 This was pushed by @Dr-G and @willingc . This was added as I tried to merge the changes. The main change was in commit https://github.com/systers/vms/pull/590/commits/2bf3a41f58ca92da176920be8332735547761ab3.

Reply:
@tonythomas01 What should I do in this case?

Reply:
I also cherry-picked my commit. But these merged commits still appeared. Thanks.

Reply:
@sammy1997 I guess you must rebase your commits.

Reply:
@anjali-dhanuka Thanks. I completely forgot to rebase my branch.

Reply:
@tonythomas01 I will do this and send a new PR.